docs/security/auth/tokenx.md

```diff
@@ -204,6 +204,7 @@ Tokendings will then issue an `access_token` in JWT format, based on the paramet
 * You have a _subject token_ in the form of an `access_token` issued by one of the following providers:
     - [ID-porten](idporten.md)
     - Tokendings
+    - [Loginservice](https://doc.nais.io/legacy/sunset/#loginservice) (Remember that loginservice is a legacy system. TokenX accept their tokens to ease migration away from on-prem.)
```

Comment:
Should use a relative link here that'll be easier to catch when refactoring.

Reply:
Har oppdatert til å bruke relativ sti. Ser ut som markdown-en deres også støtter lenker direkte til andre markdown-filer, som formodentlig blir oversatt når det konverteres til HTML. Vet ikke hva slags markdown-system dere bruker, så hadde vært best om dere bare oppdaterer lenken selv hvis dere ønsker det annerledes enn det jeg endret det til.